builders/build/templates.md

```diff
@@ -0,0 +1,130 @@
+
+## Baseline Appchain template {: #baseline-appchain-template }
+
+Developing a Substrate-based runtime typically involves two main steps: first, [incorporating pre-existing Substrate built-in modules](XXX){target=_blank} into the runtime, and second, [crafting new modules](XXX){target=_blank} tailored to the specific application.
+
+Since the template already contains the necessary configuration for seamless integration into the Polkadot ecosystem and for Tanssi protocol compatibility, teams interested in constructing an innovative Appchain can use this template as a starting point to start adding their custom logic.
+
+## Baseline EVM (Ethereum Virtual Machine) Template {: #baseline-evm-template }
+
+For teams developing their applications on top of an EVM (Ethereum Virtual machine), this template is a foundational starting point. It contains all the essential references to add the extra layer of Ethereum compatibility to a Substrate node: 
+
+- **EVM** - this referece adds the execution layer for Ethereum apps
+- **Ethereum** - this reference adds the Ethereum block production emulation to allow the RPC nodes (and DApps) to run without any modification
+- **EVMChainId** - stores the chain identifier that identifies the Ethereum network
+
+Since the template already contains the necessary configuration for seamless integration into the Polkadot ecosystem and for Tanssi protocol compatibility, if the use case is entirely developed on top of the EVM, then this template requires no additional changes in the runtime. This means that this template is ready to be built and deployed through Tanssi.
+
+## Adapting an Existing Runtime {: #adapting-existing-runtime }
+
+For teams who already have a Substrate runtime built, it will be necessary to implement the required modules and configurations into the runtime. This will ensure that the runtime can evolve into a ContainerChain that can be successfully [deployed through Tanssi](#base-setup-supporting-tanssi) and [run properly within Polkadot](#base-setup-integrating-into-polkadot).
+
+Failing to do so might lead to reduced interoperability within the ecosystem and unnecessary exposure to vulnerabilities.
+
+### Adding Cumulus Support {: #adding-cumulus-support }
+
+If the runtime is set up as a solo-chain, check on the official [Cumulus template](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/parachain-template){target=_blank} or on any of the templates available in the [Tanssi repository](https://github.com/moondance-labs/tanssi){target=_blank} for a reference setup.
+
+### Adding Tanssi Protocol Support {: #adding-tanssi-support }
+
+To support the Tanssi protocol, it will be necessary to add [the modules](#base-setup-supporting-tanssi) through the following steps:
+
+1. Include the dependencies in the `Cargo.toml` manifesto
```

Comment:
Where is the manifesto located?

Reply:
haha, I thought about it, and left it without mention because I don't know the structure of their project.
I´ll add something like ¨wich is usually located at ....¨